services/cypress/integration/thirdPartyAPIAccess/thirdPartyAPIAccess.spec.ts

```diff
@@ -69,7 +69,12 @@ describe('thirdPartyAPIAccess', () => {
                 body: '{"query":"query IndexRates { indexRates { totalCount edges { node {  id } } } }"}',
                 failOnStatusCode: false,
             }).then(res => {
-                expect(res.status).to.equal(200)
+                if (Cypress.config().baseUrl === 'http://localhost:3000') {
+                    expect(res.status).to.equal(200) // okay 
+                } else {
+                    expect(res.body.errors[0].message).to.contain('this IP address is not in the allowed list')
+                    expect(res.status).to.equal(500) // server error 
```

Comment:
I would not expect a 500 for a request that is coming from the wrong IP address. 500 should only be returned when an unexpected internal server error has occurred. An outlawed IP address should be a 403 forbidden, I think